src/test/java/com/onfido/integration/TestBase.java

```diff
@@ -54,6 +54,7 @@ public Applicant createApplicant(String first_name)
                 .lastName("Last")
                 .email("first.last@gmail.com")
                 .phoneNumber("351911111111")
+                .putAdditionalProperty(ConsentItem.NameEnum.PRIVACY_NOTICES_READ.toString(), true)
```

Comment:
What i can also do here is:

```
                .putAdditionalProperty(
                  ConsentItem.NameEnum.PRIVACY_NOTICES_READ.toString(), 
                  Map.of("granted", true)
               )
```

But it will also not work